jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/content/CachingHttpContentFactory.java

```diff
@@ -596,9 +595,9 @@ public Resource getResource()
         }
 
         @Override
-        public ByteBuffer getByteBuffer()
+        public void writeTo(Content.Sink sink, long offset, long length, Callback callback)
         {
-            return null;
+            callback.succeeded();
```

Comment:
Should this write EOF and then succeed the callback?

Reply:
Yes, I think this should. Let me change that.